### ISSUE TITLE
Please help us update the project glossary in the wiki

### ISSUE THREAD

**Issue by jodh-intel** [kata-containers/kata-containers#3176]
:warning: PR + issue free zone! :warning: :smile: 

# TL;DR

Please help update the glossary file in the wiki:

- https://github.com/kata-containers/kata-containers/wiki/Glossary

This is Christmas come early: *you do not need to raise issues or PRs to help with this effort!* :santa: :mrs_claus: :gift: :christmas_tree: :star: 

# Detail

See my original ML posting on the topic for some juicy detail:

- http://lists.katacontainers.io/pipermail/kata-dev/2021-December/002302.html

## Summary

I've copied the [Kata glossary](https://github.com/kata-containers/kata-containers/blob/main/Glossary.md) to the [Kata wiki](https://github.com/kata-containers/kata-containers/wiki/Glossary) to allow for some collaborative hacking to "fill in the blanks" in that document. Once we've got more content, we can either:

- Raise a PR to copy the wiki version back to https://github.com/kata-containers/kata-containers/blob/main/Glossary.md

  And update the wiki link to point at that file!

- Delete the in-tree version of the file and use the wiki version instead

  If we decide it will be changing a lot?

Related: https://github.com/confidential-containers/documentation/issues/13.

> **Note:** There is a small danger someone will update the in-tree version of the file whilst we hack on the wiki version, but I think the risk is infinitessimely small given there is only a single version of that file in git. I'll check when we decide which approach to follow above anyhow and update the appropriate version accordingly...

**Comment by jodh-intel** [comment]
Ping to the community to help with this important activity.

If you're keen to get started with Kata, but don't know where to start, this is the quickest and simplest way to get involved! :smile:

btw, even if you don't know the definitions, simply adding some terms to the glossary is helpful as we'll then know what we need to define.

**Comment by jodh-intel** [comment]
We've made a lot of updates to the glossary now. ptal at it and get hacking folks!

- https://github.com/kata-containers/kata-containers/wiki/Glossary
